frontend/src/Feed.js

```diff
@@ -17,14 +18,14 @@ class Feed extends React.Component {
         if (this.state.messages == null) {
             return (<div> loading messages</div>);
         }
-
-        if (this.state.messages.length === 0) {
-            return (<p> There are no posts yet </p>);
-        }
-
         return (
             <div>
+                <div>
+                    <textarea value={this.state.text} onChange={this.onTextChange.bind(this)}></textarea>
+                    <button onClick={this.onButtonClick.bind(this)} disabled={this.state.text.length === ''}> submit </button>
+                </div>
                 <div className="message-container">
+                    {this.state.messages.length === 0 && (<p> There are no posts yet </p>)}
```

Comment:
Notice how I removed the if statement above. This line will display the messae "There are no posts yet" if there are no messages to be found".